vision/cloud-client/detect/beta_snippets.py

```diff
@@ -272,7 +273,75 @@ def detect_document_features_uri(gcs_uri):
                         for symbol in word.symbols:
                             print('\t\t\tSymbol: {} (confidence: {})'.format(
                                 symbol.text, symbol.confidence))
-# [END vision_fulltext_detection_pdf_gcs_beta]
+# [END detect_batch_annotate_files_gcs_beta]
+
+
+# [START detect_async_batch_annotate_images_beta]
```

Comment:
nit: `vision_detect_async_batch_annotate_images_gcs_beta`
I feel like we usually put `gcs` in the region tag when the parameters are gcs links, but not sure if this still applies to async.